### ISSUE TITLE
Change message box to something more visual

### ISSUE THREAD

**Issue by strinsberg** [SibylLab/Program-Wars#594]
**Is your feature request related to a problem? Please describe.**
The text based message box is not working as expected. I think it would be much more informative and user friendly to make it visual.

**Describe the solution you'd like**
Have a smaller area that can display a list of the last 5-10 turns with an icon showing the card played and the player that played it (maybe even the target). This could also still have a button to open a text history that has more detail to show point amounts and targets etc. This text box could even be styled as code.

This way the player would only have to read text and scroll the message box when they wanted detailed or past info.

As an implementation this would also be easier than the current message box because we could just store every move that was made and visualize it in different ways. The current event catching and creation of text is clumsy and seems error prone.

**Additional context**
How the game Hearthstone does it...
![hearthstone](https://user-images.githubusercontent.com/45857986/85309986-ab35fb80-b470-11ea-811f-6bec78714328.jpg)
A mock up of how it might look in our game...
![visual_turn_log](https://user-images.githubusercontent.com/45857986/85310006-b1c47300-b470-11ea-9aec-9a464f748f01.png)

**Comment by strinsberg** [comment]
Here is the progress I have made on this so far. I think the visuals could be tweaked a bit, but it makes it much easier to track what is going on in the game I think.
![Screenshot from 2020-06-23 16-07-13](https://user-images.githubusercontent.com/45857986/85470126-d4878200-b56b-11ea-9c17-876a867ce6d3.png)

**Comment by strinsberg** [comment]
I have adjusted this some more and added target pictures to the images when they are necessary. They are a little small, but I think they do the trick.

Part of the reason for this was that it is still quite complicated to give the player nice text messages for the turn history, because almost every type of play requires a change to the message text. So after some experimenting I have decided not to add the textual history. Since we have game logs that can contain info we need for debugging I don't think we need this. There is enough information for players to figure out what is going on. BUT we can always add more details later if we think it is still necessary.

![Screenshot from 2020-06-24 17-15-41](https://user-images.githubusercontent.com/45857986/85636793-aaeb5b00-b63e-11ea-8d9c-8fcb67d03b24.png)
![Screenshot from 2020-06-24 17-16-20](https://user-images.githubusercontent.com/45857986/85636795-ab83f180-b63e-11ea-84b3-8e48a78164c9.png)
